docs_src/tutorial_1_hello_world.py

```diff
@@ -0,0 +1,20 @@
+from corva import Api, Corva, Event, State
+
+app = Corva()  # 1 initialize the app
+
+
+@app.stream  # 2 add decorator with needed event type to your function
+def user_job(event: Event, api: Api, state: State):
+    # 3 add parameters with predefined types, that will be injected automatically
+
+    """User's main logic function"""
+
+    pass
+
+
+def lambda_handler(event, context):
+    # 4 define function that will be run by AWS lambda
+
+    """AWS lambda handler"""
+
+    user_job.run(event)  # 5 run your function
```

Comment:
still, I think it is a bad design to expose information about internal infrastructure to the user

Reply:
do you mean `lambda_handler` or `user_job.run`?

Reply:
I mean `lambda_handler`. It exposes information about lambda usage. Everything is ok with `user_job.run` - it's just part of the user-interface and has no redundant information for a user

Reply:
Well, almost everything is ok. `user_job` is a function. It has no methods. So, this line won't work

Reply:
will make it work like this [link](https://docs.celeryproject.org/en/stable/getting-started/first-steps-with-celery.html#calling-the-task)

Reply:
I agree about `lambda_handler`, but it has to do to more with how dev center works, not sdk

Reply:
> I agree about lambda_handler, but it has to do to more with how dev center works, not sdk

It is possible to change how the dev center works

> will make it work like this link

why we cannot just call a function as it is? `user_job(event)`

Reply:
> It is possible to change how the dev center works. How hard it will be to change it on the dev center side?

Reply:
> It is possible to change how the dev center works. How hard it will be to change it on the dev center side?

@jordanambra @ville 

> why we cannot just call a function as it is? user_job(event)

we can